components/Form/Checkboxes/StyledCheckbox/StyledCheckbox.module.scss

```diff
@@ -30,16 +30,12 @@
 
     &:hover::before {
         border: 2px solid #ad8e28;
+        transition: border-color 0.2s ease-in-out;
```

Comment:
The transition only applys on hover, not when un-hovering. This transition needs to be under `&::before` not `&:hover::before`